features/install.feature

```diff
@@ -37,9 +75,47 @@ Feature: Installation
         reporter: dot
 
       # Path patten used for finding tests (see https://github.com/isaacs/minimatch)
-      testFilePattern: '**/*_{spec,test}.{coffee,js}'
+      testFilePattern: '**/*_{spec,test}.coffee'
```

Comment:
Since dashes are valid in filenames as well, and others might use camelCase filenames, we should get rid of the `_` here: `'**/*{spec,test}.coffee'`